modules/get-started/pages/whats-new.adoc

```diff
@@ -1,79 +1,95 @@
 
-== Controller snapshots
+== Whole cluster restore
 
-Redpanda now loads the controller log from a snapshot on startup. xref:./architecture.adoc#controller-partition-and-snapshots[Controller snapshots] save the current cluster metadata state to disk, which significantly improves startup times of brokers in long-running Redpanda clusters. For example, with a partition that has moved several times, a snapshot can restore the latest state without replaying every move command.
+With Tiered Storage enabled, you can do a whole cluster restore to recover data (including its metadata) from a failed cluster to a new cluster. This is a simpler and cheaper alternative to active-active replication, for example with MirrorMaker 2. Use this recovery method to restore your application to the latest functional state as quickly as possible.
 
-== Delete records from a topic
+== Recovery mode 
 
-Redpanda enables you to xref:develop:config-topics.adoc#delete-records-from-a-topic[delete data from the beginning of a partition up to a specific offset]. The offset represents the true creation time of the event, not the time when it was stored by Redpanda. Deleting records frees up space in local disk and in object storage, which is especially helpful if your producers are pushing more data than you anticipated when sizing your storage infrastructure, or if you want to implement a data retention policy aligned with a particular business event and not based on age or size. There are different ways to delete records from a topic, including using the `rpk topic trim-prefix` command or using the DeleteRecords Kafka API with Kafka clients.
+xref:manage:recovery-mode.adoc[Recovery mode] lets you repair and restore a failed cluster that cannot start normally due to issues such as a system crash or out-of-memory errors. In recovery mode, Redpanda limits functionality to cluster configuration changes and other manual administrative actions so that you can repair the cluster.
 
-== Topic-aware leadership balancing
+== Compaction improvements
 
-The new default xref:manage:cluster-maintenance/cluster-balancing.adoc#partition-leadership-balancing[`leader_balancer_mode`] property ensures that each shard in a cluster is assigned an equal number of partition leaders and attempts to spread every topic's partition leaders evenly across all brokers in a cluster.
+xref:manage:cluster-maintenance/compaction-settings.adoc[Compaction] follows a new algorithm, making it more efficient and similar to Kafka's compaction. Redpanda builds a map of keys to track the latest offset per key using multiple segments. The compaction process uses this key map to de-duplicate keys within the compacted segments. It no longer stores full-fidelity keys, but instead maps SHA-256 hashes to offsets. The amount of memory that each shard may use when creating the key map for a partition is configurable. The default is 128 MiB. 
```

Comment:
Let's also put something in here about compaction boundaries (after the first sentence).

e.g. 'Compaction boundaries are no longer created between segments or raft terms, and compaction is performed on data already uploaded to tiered storage so long as that data also still exists in local storage, improving efficiency of compaction in both storage tiers.

look good enough @andrwng ?